src/test/java/cloud/eppo/helpers/AssignmentTestCase.java

```diff
@@ -37,11 +49,138 @@ public List<SubjectAssignment> getSubjects() {
     return subjects;
   }
 
-  public void setFileName(String fileName) {
-    this.fileName = fileName;
+  private static final ObjectMapper mapper =
```

Comment:
Below is where I relocated all the useful `AssignmentTestCase` related utility methods. They are public so they can be used by upstream SDks